osu.Game/Online/Chat/NowPlayingCommand.cs

```diff
@@ -59,9 +67,31 @@ protected override void LoadComplete()
                     break;
             }
 
-            string beatmapString = beatmapInfo.OnlineID > 0 ? $"[{api.WebsiteRootUrl}/b/{beatmapInfo.OnlineID} {beatmapInfo}]" : beatmapInfo.ToString();
+            string beatmapString()
+            {
+                string beatmapInfoString = localisation.GetLocalisedBindableString(beatmapInfo.GetDisplayTitleRomanisable()).Value;
+
+                return beatmapInfo.OnlineID > 0 ? $"[{api.WebsiteRootUrl}/b/{beatmapInfo.OnlineID} {beatmapInfoString}]" : beatmapInfoString;
```

Comment:
Similarly here, making this a link is eating up even more characters, making the 100-character limit for message even more constraining. It's bad enough that on an online map it only takes the link and 2 selected mods to exceed the limit.

Reply:
Including links is the default behavior of stable, lazer used to behave like this, so the problem might be character limit

Reply:
It's an interesting problem for sure. Perhaps the solution is to make a separate POST request to osu-web that has more relaxed limits for this particular case.

Reply:
100 characters seems potentially too low. bancho is set to 450 characters, and even if the message exceeds that it is truncated, not rejected. Maybe worth opening an osu-web issue to track this?

Reply:
It seems to be controlled by the osu-web .env config `CHAT_MESSAGE_LENGTH_LIMIT`.

Reply:
Yeah, aware of that. I guess we can just set it up to 450 for now as a first step. I'll do that.

Reply:
It looks to already be 450 on production. Could this just be a staging thing?

Reply:
I've updated staging to match production.